proposals/Proposal Code TEMPLATE

```diff
@@ -0,0 +1,27 @@
+File title should be: "[RGP-XX] Proposal Title"
+________________________________________________________
+
+# [RGP-XX] Proposal Title
+
+If executed, this proposal will:
+
+1. 1-2 sentences describing the on-chain action that would be triggered by the passing of this proposal.
+
+**How was this proposal discussed?**
+Please see discussions here:
+* Discussion: link to Discussion forum post
+* Formal Review: link to Formal Review forum post 
+* Snapshot Poll: link to Snapshot poll 
+
+## Notes
+* Add any relavant token contracts, the Radworks Treasury address if requesting funds from the treasury, and the receiving address as well as who it belongs to
+* e.g. `0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48` is the USDC contract (https://etherscan.io/token/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48).
+* e.g. `0x8da8f82d2bbdd896822de723f55d6edf416130ba` is the Radworks Treasury address.
+
+
+## ACTIONS ##
+```
+[Insert token contract] 0 "transfer(address,uint256)" [Instert receiving address] [Insert amount]
+
+Ex: 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48 0 "transfer(address,uint256)" 0xaddressxxxxxxxxxxx 1000000
+```
```

Comment:
```suggestion
Ex: 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48 0 "transfer(address,uint256)" 0xaddressxxxxxxxxxxx 1000000
```
*NOTE: Please always double check the addresses in the proposal code before copy and pasting.*
```